src/categorical_algebra/FinCats.jl

```diff
@@ -220,6 +219,7 @@ compose(C::FinCatPathGraph, fs...) =
   reduce(vcat, coerce_path(graph(C), f) for f in fs)
 
 hom(C::FinCatPathGraph, f) = coerce_path(graph(C), f)
+hom(C::FinCatPathGraph, f::Nothing) = nothing
```

Comment:
Ditto for this line, and several others below. Please check this.

Reply:
Oops, yeah, these were probably from a draft that used nothing where I really ended up using GatExpr{:nothing}